provisioner/docker/docker-compose.yml

```diff
@@ -13,6 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+version: "2"
```

Comment:
Do you verify it for other OS (CentOS, Rockylinux, Debian or Ubuntu)?

Reply:
>> docker compose yml file should add version: "2", otherwise can't create the docker env to smoke test.

Why haven't the other distros encountered this issue?

Reply:
> > > docker compose yml file should add version: "2", otherwise can't create the docker env to smoke test.
> 
> Why haven't the other distros encountered this issue?

It will failed when we miss the _**version "2"**_ in openeuler.
![image](https://github.com/apache/bigtop/assets/11262916/fe08834c-5bad-4014-9df0-c7efa5b66464)

Reply:
> Do you verify it for other OS (CentOS, Rockylinux, Debian or Ubuntu)?

Other OS is ok, i will continue to analyze the problem.

Reply:
You should upgrade the docker-compose to 2.x:
```
docker-compose -v
Docker Compose version v2.6.0
```

Reply:
@guyuqi I upgrade the docker-compose to 2.6.0, it's ok without  _**version: "2"**_

![image](https://github.com/apache/bigtop/assets/11262916/8b26a146-981b-4f9c-a712-eeaa99c13550)